src/gui_impl/main_window.cpp

```diff
@@ -44,7 +44,7 @@ MainWindow::MainWindow(QSharedPointer<ApplicationContext> application_context)
     setUnifiedTitleAndToolBarOnMac (true);
     
     QSettings settings("SigViewer");
-    resize(settings.value("MainWindow/size", QSize(900, 800)).toSize());
+    resize(settings.value("MainWindow/size", QSize(1500, 800)).toSize());
```

Comment:
@cbrnr Sure thing, will make that change soon. In the meantime, somehow the close file button is gone: I thought it might be useful. Should we add it back?
 
![2017-03-06 4 _li](https://cloud.githubusercontent.com/assets/20526646/23630563/9c179e84-0289-11e7-856a-ad3efa5c76f1.jpg)